book/src/ecosystem/the-composable-team.md

```diff
@@ -0,0 +1,94 @@
+# The Composable Team
+
+---
+
+Composable Finance brings together a well-rounded team of individuals who are fully immersed in the blockchain and decentralized finance spaces. Through their varied experiences, the Composable team has come to recognize that the future of blockchain is cross-chain and cross-layer. Under this shared mission, our team unites, bringing with them all of the tools and skills needed to break down the barriers dividing the blockchain industry. 
+
+### Cosmin Grigore | CEO 
+
+Cosmin entered the crypto space in 2017, initially starting with Solidity development and trading. He has had a long career in computer science, working as a full-stack engineer before pivoting into engineering management. Cosmin made a full transition to crypto in 2020, working as a dev lead for [nakamo.to](https://www.nakamo.to/), an [Advanced Blockchain AG](https://www.advancedblockchain.com/) portfolio company. While there, he had the idea for Composable Finance, and started working with Advanced Blockchain to make it a reality. Cosmin is Composable’s venerable leader, bringing extensive experience and cutting-edge ideas, while managing a complex product and ecosystem. 
+
+### 0xbrainjar | Head of Product 
+
+Our Head of Product is currently anonymous. Originally trained in physics, he brings valuable DeFi know-how and passion. Check him out on [Twitter](https://twitter.com/0xbrainjar) or [Medium](https://0xbrainjar.medium.com/) for product updates and industry insights. He’s also leading [Composable Labs](https://0xbrainjar.medium.com/introducing-composable-labs-an-incubator-and-testing-ground-for-projects-using-composables-tech-b467d438142c), the incubator and the testing ground for projects using Composable’s tech stack. 
+
+### Karel Kubat | CTO 
+
+Karel is a long-term crypto user and developer, passionate about decentralization through technological means. Karel has worked on a range of protocols and contracts, such as Ethereum, NEO, Substrate, and custom chains. He brings this expertise to the Composable team, working to help democratize financial instruments and markets.
+
+### Zain | CMO 
+
+Zain is a seasoned art, brand and marketing director who has worked between the intersection of tech and development. Previously head of brand and communications at Biconomy, marketing head at Muzmatch (Y Culimantor), Zain has built award-winning campaigns and marketing strategies for his innovative organizations and businesses. He brings a wealth of industry and related experience to the role. 
```

Comment:
Y Combinator?

Reply:
hmm it's also Y Culimantor on the original page

Reply:
changed to Y Combinator